src/components/Book/Book.tsx

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+import "./Book.scss";
+import { BookType } from "../BookList/BookList";
+
+export const Book = (props: BookType) => {
+  const [bookImage, setBookImage] = useState("./book-cover-placeholder.png");
```

Comment:
you created state for static url to image? Please check how we insert image src in React
https://stackoverflow.com/questions/37644265/correct-path-for-img-on-react-js